docs/source/tutorial/spaceflights_tutorial.md

```diff
@@ -1,20 +1,20 @@
 # Kedro spaceflights tutorial
 
-**Scenario**: *It is 2160 and the space tourism industry is booming. Globally, there are thousands of space shuttle companies taking tourists to the Moon and back. You have been able to source amenities offered in each space shuttle, customer reviews and company information.*
+**Scenario**: *It is 2160 and the space tourism industry is booming. Globally, thousands of space shuttle companies take tourists to the Moon and back. You have been able to source amenities offered in each space shuttle, customer reviews and company information.*
 
-**Project**: *You want to construct a model for predicting the price for each trip to the Moon and the corresponding return flight.*
+**Project**: *You want to construct a model that predicts the price for each trip to the Moon and the corresponding return flight.*
 
 In this tutorial, we illustrate the typical Kedro workflow and the steps necessary to convert an empty Kedro project template into a working project.
 
-In the text, we assume that you create an empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately 2 hours and you will learn each step of the Kedro project development workflow, by working on an example to construct nodes and pipelines for the price-prediction model.
+In the text, we assume that you create an empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately two hours, during which time you will learn each step of the Kedro project development workflow, by working on an example to construct nodes and pipelines for the price-prediction model.
 
 ```{note}
-You may prefer to get up and running more swiftly so we provide the full spaceflights example project as a [Kedro starter](../get_started/starters.md). To create the project, run `kedro new --starter=spaceflights`. When prompted for a project name, enter `Kedro Tutorial`. Subsequently, accept the default suggestions for `repo_name` and `python_package` by pressing enter. This will generate a project from the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) so you can follow the tutorial without any of the copy/pasting.
+You might prefer to get up and running more swiftly, so we provide the full spaceflights example project as a [Kedro starter](../get_started/starters.md). To create the project, run `kedro new --starter=spaceflights`. When prompted for a project name, enter `Kedro Tutorial`. Subsequently, click enter to accept the default suggestions for `repo_name` and `python_package`. This will generate a project from the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) so you can follow the tutorial without having to copy or paste.
```

Comment:
```suggestion
You might prefer to get up and running more swiftly, so we provide the full spaceflights example project as a [Kedro starter](../get_started/starters.md). To create the project, run `kedro new --starter=spaceflights`. When prompted for a project name, enter `Kedro Tutorial`. Subsequently, press enter to accept the default suggestions for `repo_name` and `python_package`. This will generate a project from the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights) so you can follow the tutorial without having to copy or paste.
```